docker-compose.yml

```diff
@@ -6,3 +6,5 @@ services:
     environment:
       POSTGRES_PASSWORD: postgres
       POSTGRES_DB: clines
+    ports:
+      - "5432:5432"
```

Comment:
I did this once, but we ended up solving with another thing, is this opening the port to see the content of postgres?